tests/docker/ducktape_cluster.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
should this be generated instead of hard coded? 

either by some bash script or some python script ?

Reply:
yeah, a jinja2 template or something alike... coming up in a followup PR.